packages/cozy-ccc-libs/src/index.js

```diff
@@ -1,5 +1 @@
-import { Bridge, MessengerInterface } from './bridge/bridgeInterfaces'
-import ContentScript from './ContentScript'
-import LauncherLib from './launcher'
-
-export { Bridge, MessengerInterface, ContentScript, LauncherLib }
+export { default as ContentScript } from './ContentScript'
```

Comment:
We don't export anymore Bridge, MessengerInterface and LauncherLib? Is is intended? Can you explain why in a dedicated commit?